<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QLineEdit
{
border-radius: 5px;
border: 2px solid #CCC;
}
QScrollArea
{
border-radius: 10px;
border: 2px solid #DDD;
}

QPushButton 
{
color: #111;
background-color: rgba(255, 255, 255, 0);
border-radius: 5px;
border: 2px solid #555;
padding-left:8px;
padding-right:8px;
padding-top:2px;
padding-bottom:2px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="main_widget" native="true">
     <property name="styleSheet">
      <string notr="true">#main_widget {
                         background: qlineargradient(
                            spread:pad,
                            x1:0, y1:1, x2:1, y2:1,
                            stop:0 #ff00ff,
                            stop:1 #00ff00
                        )
                     }
                     QWidget, QLabel, QPushButton {
                        color: #ffffff;
                     }</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="horizontalWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,0,0">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="line_edit_search">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit	{
	background-color: rgba(252, 252, 252, 120);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Введите текст для поиска ...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="but_search">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(252, 252, 252, 160);</string>
           </property>
           <property name="text">
            <string>Поиск</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../../.designer/icons/search_2.png</normaloff>../../../../../.designer/icons/search_2.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="but_is_filter">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(252, 252, 252, 160);</string>
           </property>
           <property name="text">
            <string>Все фильтры</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../../.designer/icons/settings_4.png</normaloff>../../../../../.designer/icons/settings_4.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_filter" native="true">
        <layout class="QVBoxLayout" name="layout_filter">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="check_box_hide_learned_words">
              <property name="text">
               <string>Скрыть выученные слова</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="combo_box_sort">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">color:#000;</string>
              </property>
              <item>
               <property name="text">
                <string>Без сортировки</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Правильных ответов ↑</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Правильных ответов ↓</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="check_box_show_hard_words_only">
              <property name="text">
               <string>Показать тяжелые слова</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="but_clear_filter">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton 
{
background-color: rgba(252, 252, 252, 160);
border: 1px solid #888;
padding-left:4px;
padding-right:4px;
padding-top:2px;
padding-bottom:2px;
}</string>
              </property>
              <property name="text">
               <string>Сброс фильтров</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scroll_area">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QScrollArea{
background-color: rgba(252, 252, 252, 0);
}
QScrollBar:vertical {
    background: transparent;
    width: 8px;
    margin: 0px;
}
QScrollBar::handle:vertical {
    background: #888;
    border-radius: 4px;
}
QScrollBar::handle:vertical:hover {
    background: #555;
}
QScrollBar::add-line, QScrollBar::sub-line {
    height: 0px;
}</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="widget_list">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>550</width>
           <height>295</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget {
background-color: rgba(252, 252, 252, 0);
}</string>
         </property>
         <layout class="QVBoxLayout" name="layout_scroll">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_count">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel	{
	border-radius: 8px;
	padding-top: 5px;
	padding-bottom: 5px;
	padding-left: 15px;
	padding-right: 15px;
	border: none;
	background-color: rgba(252, 252, 252, 120);
}</string>
           </property>
           <property name="text">
            <string>Количество: 0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
